pom.xml

```diff
@@ -45,6 +45,7 @@
 
   <org.apache.commons.lang3.version>3.9</org.apache.commons.lang3.version>
   <org.apache.commons.codec.version>1.13</org.apache.commons.codec.version>
+  <org.apache.commons.validator.routines.version>1.6</org.apache.commons.validator.routines.version>
```

Comment:
I think this property should just be named `org.apache.commons.validator.version`, no? The `routines` is part of the specific package you're importing something from right now, but not part of the artifact name. Well, even the `org.apache` isn't part of the artifact name, but still.

Reply:
Ah, yeah that makes sense...I wasn't too sure so I just copied it from the import statement haha.